src/checkstyle/checkstyle-suppressions.xml

```diff
@@ -20,6 +20,10 @@
 	<suppress files="[\\/]spring-boot-docs[\\/]" checks="JavadocType|JavadocVariable" />
 	<suppress files="[\\/]spring-boot-docs[\\/]" checks="SpringJavadoc" message="\@since" />
 	<suppress files="[\\/]spring-boot-docs[\\/].*jooq" checks="AvoidStaticImport" />
+
+	<!-- Can be removed after https://github.com/spring-io/spring-javaformat/pull/316 has been merged and released -->
+	<suppress files="[\\/]spring-boot-docs[\\/].*MyRoutingConfiguration\.java" checks="AvoidStaticImport"/>
```

Comment:
Rather than adding a comment for this sort of thing, we prefer to open an issue. We find it's less likely to get forgotten when handled that way.